File: base/src/main/java/de/whitefrog/frogr/exception/FrogrException.java
package de.whitefrog.frogr.exception;

public class FrogrException extends RuntimeException {
    public FrogrException() {
        super();
    }

    public FrogrException(String message) {
        super(message);
    }

    public FrogrException(String message, Throwable cause) {
        super(message, cause);
    }

    public FrogrException(Throwable cause) {
        super(cause);
    }

    protected FrogrException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
        super(message, cause, enableSuppression, writableStackTrace);
    }
}

File: base/src/main/java/de/whitefrog/frogr/exception/FrogrException.kt
package de.whitefrog.frogr.exception

open class FrogrException : RuntimeException {
  constructor() : super()

  constructor(message: String?) : super(message)

  constructor(message: String?, cause: Throwable) : super(message, cause)

  constructor(cause: Throwable) : super(cause)

  protected constructor(message: String, cause: Throwable, enableSuppression: Boolean, writableStackTrace: Boolean) : super(message, cause, enableSuppression, writableStackTrace)
}